openfasoc/generators/ldo-gen/flow/scripts/openfasoc/README.md

```diff
@@ -0,0 +1 @@
+Scripts not in OpenROAD-flow-scripts, created for use within the OpenROAD flow in OpenFASOC.
```

Comment:
Not sure what you meant here?

Reply:
These scripts are not part of default ORFS. They were created for OpenFASOC